packages/kbn-optimizer/limits.yml

```diff
@@ -80,7 +80,7 @@ pageLoadAssetSize:
   uiActions: 97717
   uiActionsEnhanced: 313011
   upgradeAssistant: 81241
-  uptime: 40825
+  uptime: 232989
```

Comment:
Sorry, but I can't imagine that a 200k increase is justified here, have you read through https://www.elastic.co/guide/en/kibana/current/ci-metrics.html#ci-metric-resolving-overages?

Reply:
Agreed. This is the result of 

```sh
$ node scripts/build_kibana_platform_plugins.js --update-limits
```

I'll read through the docs and see if I can get this limit in a more reasonable spot.

Reply:
I reverted `limits.yml` back to the main branch and re-ran through the overage resolution process. No overages were reported, so I assume there was some intermediate commit that blew up this bundle unintentionally. 

Bundle size check succeeded in latest run: https://github.com/elastic/kibana/pull/103554/checks?check_run_id=2953859894